bites.py

```diff
@@ -134,9 +139,9 @@ def take_all_turns(self):
     self.render_game()
     while 1:
       for player in self.players:
-        (self.trail, self.ant_positions, self.anthill, self.anthill_food_tokens) = \
+        (self.trail, self.ant_positions, self.anthill, self.anthill_order, self.anthill_food_tokens) = \
```

Comment:
`Player.take_turn()` does not return the `anthill_order` (rightfully so) but you've included it here. How is this not causing problems?
```suggestion
        (self.trail, self.ant_positions, self.anthill, self.anthill_food_tokens) = \
```